business/workloads.go

```diff
@@ -979,7 +980,7 @@ func (in *WorkloadService) fetchWorkloadsFromCluster(ctx context.Context, cluste
 			Pods:     models.Pods{},
 			Services: []models.ServiceOverview{},
 		}
-		w.Cluster = cluster
+		w.Namespace = models.Namespace{Cluster: cluster, Name: namespace}
```

Comment:
@jshaughn and I recently touched on this same thing for services. The problem parsing a `Namespace` this way is that you remove all the other `Namespace` fields that exist on the object: https://github.com/kiali/kiali/blob/76c4ec1806180ac7dc5a7874f7a8cb8f1103b0ef/models/namespace.go#L29-L45

For example `namespace.IsAmbient` would return different results than `workload.namespace.IsAmbient` even though they represent the same namespace. Most of the other models just have `namespace` as a string and I think that's all you need in most cases if `cluster` is also on the model. We should be consistent either way.

Reply:
right, will rework these parts also in apps